### ISSUE TITLE
should have a process that validate committee meetings data

### ISSUE THREAD

**Issue by OriHoch** [hasadna/Open-Knesset#585]
after #572 committee meetings data of new meetings should be correct

we should also have a process that validates existing meetings data and makes sure that old meetings data is correct as well

**Comment by OriHoch** [comment]
currently, it's quite feasible to run it manually, so will postpone this to later milestone
